.circleci/config.yml

```diff
@@ -544,6 +544,15 @@ on-integ-branch: &on-integ-branch
     tags:
       ignore: /.*/
 
+on-integ-branch-cron: &on-integ-branch-cron
+  filters:
+    branches:
+      only:
+        - master
+        - /^\d+\.\d+.*$/
+        - /^feature-.*$/
+        - /^perf-.*$/
```

Comment:
why nightly on this branches?

Reply:
Integration branches need to be built periodically to verify they don't break and turn against us at the wrong moment.
Those branches are not being pushed to very often (unless we CP on time) and this causes big gaps between builds.
Maybe we can exclude features and performance branches, but this will come with the cost of making CI logic not intuitive (that is, `on-integ-branch` != `on-integ-branch-cron`).
I suggest to leave it as-is, as we don't have too many feature- and pref- branches.

Reply:
I think this will add too much redundant nightly builds